input/assets/js/main.js

```diff
@@ -205,6 +205,34 @@ $("ul#bzoom").each(function(index, ul) {
   });
 });
 
+
+// Lazy loading for images
+var loadImage = function(self, imgSource) {
+  self.hide();
+  if (self.is("img")) {
+    self.attr("src", imgSource);
+  } else {
+    self.css("background-image", "url('" + imgSource + "')");
+  }
+  self.fadeIn("slow");
+  return self.removeClass("img-lazy").removeAttr("data-original");
+};
+
+var placeholder = "data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==";
+$(".img-lazy").each(function() {
```

Comment:
Good catch, I didn't realize there's no global onready for the document.
Fix: https://github.com/sphereio/commercetools-sunrise-design/commit/d7ae9e1ace120bec9bb9ad5cb0cd0749a0e0c1e0?diff=split